### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by rfichtner**
Some cleanup before we bump the jdbc driver

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -160,10 +156,7 @@
 						</goals>
 						<phase>process-sources</phase>
 						<configuration>
-							<roots>
```
Comment (AB-xdev) [inline pom.xml:163]: Why was this removed?

**Review by AB-xdev — CHANGES_REQUESTED** [review]
Can we however add a guide how to import checkstyle to the Readme.md?

File: pom.xml (hunk 1 above)
Comment (rfichtner) [reply pom.xml:163]: The default just takes all sources. The root-tag was a copy-paste left-over.

**Comment by rfichtner** [comment]
> Can we however add a guide how to import checkstyle to the Readme.md?

@AB-xdev Checkstyle runs with every build. In this project, the build fails if there are checkstyle violations. Are you proposing a guide to import the checkstyle config to an IDE?
